src/containers/shared/types.ts

```diff
@@ -0,0 +1,18 @@
+export interface IssuedCurrency {
+  currency: string
+  issuer: string
+}
+
+export interface IssuedCurrencyAmount extends IssuedCurrency {
+  value: string
+}
+
+export type Amount = IssuedCurrencyAmount | string
+
+export type ExplorerAmount =
```

Comment:
How is `ExplorerAmount` different than `Amount`?

Reply:
ExplorerAmount has been converted to a number